### PR TITLE
cm bugbash spellcheck edition

### PR DESCRIPTION
**Pull request by PhantomEpicness**
## About The Pull Request

Minor spellchecks found on discord. Currently trying to find more to bundle in.

## Why It's Good For The Game
social credit

## Changelog

:cl: TotalEpicness
spellcheck: Fixed several typos. Immersion restored.
/:cl:

### CODE REVIEW

**Comment by PhantomEpicness** [comment]
> You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

ratio

**Review by Agameofscones — APPROVED** [review]
thanks for saving us from this immersion breaking bug. 😩 

approved.

**Comment by stanalbatross** [comment]
I reccommend searching up commonly mispelt words

**Comment by hotimep** [comment]
immersion restored.....

File: code/modules/projectiles/guns/shotguns.dm [hunk 1]
```diff
@@ -279,7 +279,7 @@ can cause issues with ammo types getting mixed up during the burst.
 	gauge = "20g"
 
 /obj/item/weapon/gun/shotgun/combat/guard
-	desc = "The Weyland-Yutani MK221 Shotgun, a semi-automatic shotgun with a quick fire rate. Equipped with a red handle to signify its use with Military Police Honor Gaurds"
+	desc = "The Weyland-Yutani MK221 Shotgun, a semi-automatic shotgun with a quick fire rate. Equipped with a red handle to signify its use with Military Police Honor Guards"
```
Comment (Geevies) [inline code/modules/projectiles/guns/shotguns.dm:282]:
```suggestion
	desc = "The Weyland-Yutani MK221 Shotgun, a semi-automatic shotgun with a quick fire rate. Equipped with a red handle to signify its use with Military Police Honor Guards."
```

**Review by Geevies — CHANGES_REQUESTED** [review]

**Comment by Geevies** [comment]
there's also one prae ability where it says "dashs" instead of "dashes"

**Comment by PhantomEpicness** [comment]
> there's also one prae ability where it says "dashs" instead of "dashes"

I am having trouble finding this

**Review by Geevies — APPROVED** [review]

**Pull request merged**